src/aiClients/__tests__/useStateManager.test.ts

```diff
@@ -105,6 +105,33 @@ jest.mock('../useRhelLightSpeedManager', () => ({
   })),
 }));
 
+jest.mock('../useVaManager', () => ({
```

Comment:
Not sure why the mock is here if no tests were added/changed? Was there something broken?

Reply:
The tests were erroring on this new line https://github.com/RedHatInsights/astro-virtual-assistant-frontend/pull/284/files#diff-754d05124b57ba21af25e131f2e70c7f107a4d1a01f9e86b6dc061f27bf68781R27

Instead of mocking the vaManager, ill mock the getClient function :smile: